doc/man1/openssl-enc.pod.in

```diff
@@ -256,6 +256,15 @@ All RC2 ciphers have the same key and effective key length.
 
 Blowfish and RC5 algorithms use a 128 bit key.
 
+OpenSSL 3.0 changed the way the B<-S> option operates to not include the
```

Comment:
This seems a bit convoluted.. What about this?

For encryption purposes specifying the -salt option without the -S option will write the randomly generated salt to the encrypted file. If you use -S then the supplied salt is not written to the encrypted file. Note that prior to OpenSSL 3.0 the -S option would write to the encrypted file.

For decryption purposes if the salt used for encryption is not present in the encrypted file then it must be supplied using the -S option.

Reply:
Sadly, I don't find the above to be an improvement. But I think progress is possible.

```
Please note that OpenSSL 3.0 changed the effect of the B<-S> option.
Any explicit salt value specified via this option is no longer prepended to the
ciphertext when encrypting, and must again be explicitly provided when decrypting.
Conversely, when the B<-S> option is used during decryption, the ciphertext
is expected to not have a prepended salt value.

When using OpenSSL 3.0 or later to decrypt data that was encrypted with an
explicit salt under OpenSSL 1.1.1 do not the B<-S> option, the salt will then be
read from the ciphertext.
To generate ciphertext that can be decrypted with OpenSSL 1.1.1 do not use
the B<-S> option, the salt will be then be generated randomly and prepended
to the output.
```

Reply:
I hope you added the missing "use" in the 2nd line of the 2nd paragraph: "... OpenSSL 1.1.1 do not [use] the B<-S>..."